themes/digital.gov/layouts/partials/core/get_related.html

```diff
@@ -119,13 +127,11 @@ <h3>Get Started Building</h3>
   {{- $src := .src -}}
   {{- with .item -}}
     {{ $path := "" }}{{ with .File }}{{ $path = .Path }}{{ else }}{{ $path = .Path }}{{ end }}
-    <div class="promo" data-edit-this="{{- $path -}}">
+    <li class="promo" data-edit-this="{{- $path -}}">
       <div>
```

Comment:
If this div on line 131 is necessary for styling we should add a class to it. Otherwise let's remove and move styles to the `promo` class.